makefiles/suit.v4.inc.mk

```diff
@@ -16,18 +16,38 @@ export CFLAGS += -DSOCK_URLPATH_MAXLEN=128
 SUIT_VENDOR ?= "riot-os.org"
 SUIT_SEQNR ?= $(APP_VER)
 SUIT_DEVICE_ID ?= $(BOARD)
-SUIT_KEY ?= secret.key
-SUIT_PUB ?= public.key
-SUIT_PUB_HDR ?= public_key.h
+
+#
+# SUIT encryption keys
+#
+ifeq (1, $(RIOT_CI_BUILD))
+  SUIT_KEY_DIR ?= $(BINDIR)
+else
+  SUIT_KEY_DIR ?= $(RIOTBASE)/keys
+endif
+
+SUIT_KEY ?= $(SUIT_KEY_DIR)/secret.key
+SUIT_PUB ?= $(SUIT_KEY_DIR)/public.key
+
+SUIT_PUB_HDR = $(BINDIR)/riotbuild/public_key.h
+CFLAGS += -I$(dir $(SUIT_PUB_HDR))
```

Comment:
`$(dir $(SUIT_PUB_UDR))` is used twice. Maybe put this in a variable `$(SUIT_PUB_HDR_DIR)` ?

Reply:
Did you misspell SUIT_PUB_UDR wrong two times to make a point? ;) Will add.

Reply:
done